beacon-chain/p2p/broadcaster.go

```diff
@@ -369,7 +370,7 @@ func (s *Service) internalBroadcastDataColumn(
 	}
 
 	// Increase the number of successful broadcasts.
-	blobSidecarBroadcasts.Inc()
+	dataColumnSidecarBroadcasts.Inc()
```

Comment:
This did not exactly match other parts tracking pattern (they track broadcasts with no peers instead of successful ones), keeping it here as is. But let me know if no peer metric is needed